setup.cfg

```diff
@@ -16,7 +16,7 @@ classifiers =
 py_modules = pytest_sentry
 install_requires =
     pytest
-    sentry-sdk
+    sentry-sdk==2.0.0rc4
```

Comment:
does this work with `>=2.0.0rc4` ?

Reply:
This version will work with sentry-sdk>=2.0.0 including all the alpha (2.0.0a1, 2.0.0a2) and release candidate versions (2.0.0rc1-2.0.0rc4). I tried to match all this in the other PR but could not get it right (see commits about matching here: https://github.com/getsentry/pytest-sentry/pull/31)

Can you show us how to match all those versions correctly?

Reply:
Okay, we have fixed it now